flexmeasures/cli/jobs.py

```diff
@@ -110,7 +127,11 @@ def parse_queue_list(queue_names_str: str) -> List[Queue]:
     """
     q_list = []
     for q_name in queue_names_str.split("|"):
-        if q_name in app.queues:
+        if allow_failed and q_name == "failed":
+            q_list.append(
+                Queue(connection=app.queues["forecasting"].connection, name="failed")
```

Comment:
This doesn't work for me in practice. Have you checked that this is able to remove jobs from a non-empty queue?

Reply:
It seems that a preferred alternative to appending the failed queue in various places is to add the failed queue to the app's queue listing in `app.py` (lines 72 and 88).

Reply:
> I decided to not add failed queues to the app's queue listing, as that might lead to unexpected consequences somewhere else.

I agree.